independent-publisher-connectors/NFTMania/MintNFT.swagger.json

```diff
@@ -0,0 +1,84 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "MintNFT",
+    "description": "NFT Mania provides you with an Ultimate NFT experience and lets you develop applications in no time!",
+    "version": "1.0"
+  },
+  "host": "api.nftport.xyz",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/v0/mints/easy/urls": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
```

Comment:
@shreyan1999, empty Schema is not allowed, please define a proper schema or you can delete the empty schema.